File: 0001-two-sum/0001-two-sum.py
#Here the code starts
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        x = []
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[i] + nums[j] == target:
                    if len(x) < 2:
                        x.append(i)
                        x.append(j)
                        break
                else:
                    continue
        return x

File: 0001-two-sum/0001-two-sum.java
import java.util.*;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> h=new HashMap<>();
        int[] x=new int[2];
        for(int i=0;i<nums.length;i++){
            int res=target-nums[i];
            if (h.containsKey(nums[i])){
                x=new int[]{h.get(nums[i]),i};
                return x;}
            else{
                h.put(res,i);

            }
        }
        return x;

    }
}